public/app/core/specs/backend_srv.test.ts

```diff
@@ -745,4 +745,115 @@ describe('backendSrv', () => {
       });
     });
   });
+
+  describe('sanitizePath functionality', () => {
+    describe('when sanitizePath is enabled in options', () => {
+      it.each(['get', 'post', 'put', 'patch', 'delete'] as const)(
```

Comment:
Maybe we should add tests for calling `request` and `fetch` just in case code would divert at some point in the future?